### PR TITLE
Fix telemetry event re-registration in the middle of an existing Firefox session

### PR DESCRIPTION
**Pull request by jimporter**
As discussed on Slack. Here's a quick fix that just makes our telemetry category versioned. This will break the other telemetry PR, so I'll land this after and fix stuff up myself.

### CODE REVIEW

**Comment by jimporter** [comment]
I *think* the test failures are because I changed the message passing API to no longer include the `"lockbox"` category. I'm also talking with the telemetry team to see if there's a better way to do all this.

**Comment by jimporter** [comment]
@linuxwolf this should be ready for re-review.

File: src/bootstrap.js [hunk 1]
```diff
@@ -8,85 +8,102 @@ const { utils: Cu } = Components;
...
+        objects: ["manage"],
+        extra_keys: ["fxauid"],
+      },
+    });
+  } catch (e) {
+    if (e.message === "Attempt to register event that is already registered.") {
+      // eslint-disable-next-line no-console
+      console.log("telemetry events already registered; skipping registration");
+    } else {
+      throw e;
```
Comment (linuxwolf) [inline src/bootstrap.js]: I wonder if we should stop the extension from running if there are other telemetry problems.

The lack of telemetry will make @irrationalagent sad, but I don't think it's worth breaking users.  Definitely log *why*, but not sure it should stop the extension from running ...

Comment (jimporter) [reply src/bootstrap.js]: If there's another problem, it's (almost) certainly programmer error. The most common one would be if we used an invalid string for one of our events.

**Pull request merged**